frame/elections-phragmen/src/lib.rs

```diff
@@ -2458,10 +2551,8 @@ mod tests {
 					..
 				}
 			);
-			matches!(
-				unwrapped_error.post_info.actual_weight,
-				Some(x) if x < <Test as frame_system::Config>::BlockWeights::get().max_block
```

Comment:
why did you remove this matches ? you could actually keep them ?

Also we need to assert on these matches otherwise they are useless and don't test anything.

Reply:
I added an assertion to the exact value. Checking that it is less than max block is no longer needed.

Reply:
ok, also the other matches should be asserted https://github.com/paritytech/substrate/pull/8211